docs/scripts_md/ExitCodes.md

```diff
@@ -54,11 +54,11 @@ Below is a list of the possible exit codes:
 9\. Exit codes from uploadNeuroDB/NeuroDB/ImagingUpload.pm (exit codes from
 170 to 179)
 
-11\. Exit codes from uploadNeuroDB/minc\_insertion.pl (exit codes from 180 to 189)
+10\. Exit codes from uploadNeuroDB/minc\_insertion.pl (exit codes from 180 to 189)
```

Comment:
I am not sure to understand why these appears in your PR. Maybe it needs rebase?

Reply:
NO CLUE. I kept wondering and wondering. I merged aces/minor and it shows no changes... anyway I did hard reset it so it no longer shows in the diff.

Reply:
Ohh I think I know. The last one that got merged onto minor did not have the updated `.md` regenerated, so now that I removed the `## -- FROM` as per your earlier comment and regenerated the `/.md` it shows up again; so it is a legitimate change